CHANGELOG.md

```diff
@@ -30,58 +30,69 @@ This release has an [MSRV][] of 1.75.
 
 ### Added
 
-- Added `vello_svg::Error`, which is returned by new functions that read text into a `usvg::Tree`.
-- Added `vello_svg::render`, which takes an svg string and renders to a new vello scene.
-- Added `vello_svg::append`, which takes an svg string and renders to a provided vello scene.
-- Added `vello_svg::append_with`, which takes an svg string and renders to a provided vello scene with and error handler.
-- Added `vello_svg::render_tree`, which takes a usvg::Tree and renders to a provided vello scene with and error handler.
+- Added `vello_svg::Error`, which is returned by new functions that read text into a `usvg::Tree`. ([#18] by [@simbleau])
+- Added `vello_svg::render`, which takes an svg string and renders to a new vello scene. ([#18] by [@simbleau])
+- Added `vello_svg::append`, which takes an svg string and renders to a provided vello scene. ([#18] by [@simbleau])
+- Added `vello_svg::append_with`, which takes an svg string and renders to a provided vello scene with and error handler. ([#18] by [@simbleau])
+- Added `vello_svg::render_tree`, which takes a usvg::Tree and renders to a provided vello scene with and error handler. ([#18] by [@simbleau])
 
 ### Changed
 
-- Updated to vello 0.2
-- Updated to usvg 0.42
-- Renamed `render_tree` to `append_tree`
-- Renamed `render_tree_with` to `append_tree_with` and removed the `Result<(), E>` return type for the error handler.
+- Updated to vello 0.2 ([#18] by [@simbleau])
+- Updated to usvg 0.42 ([#18] by [@simbleau])
```

Comment:
```suggestion
- Updated to Vello 0.2.0 and `usvg` 0.42 ([#18] by [@simbleau])
```
or 
```suggestion
- Updated to `vello` 0.2.0 and `usvg` 0.42 ([#18] by [@simbleau])
```

Personally, I prefer the former, but I think the latter more strictly follows our current scheme.
I suspect that the "referring to the crates.io name" is not a case that we should have.

Reply:
Discussion in https://xi.zulipchat.com/#narrow/stream/419691-linebender/topic/Linebender.20capitalization.20scheme